src/Template/Bake/Template/index.ctp

```diff
@@ -39,20 +39,21 @@ $fields = collection($fields)
     endforeach;
 %>
     </ul>
-</div>
-<div class="<%= $pluralVar %> index large-10 medium-9 columns">
+</nav>
+<div class="<%= $pluralVar %> index large-9 medium-8 columns" id="content">
```

Comment:
> In rare cases you want something to "look" like <h2> but actually be just a <p>

Giving the user one impression of the markup, yet giving a html parser another is a shady practice - also not particularly relevant here.